tasks/configure-pipeline-local-fs.yml

```diff
@@ -0,0 +1,140 @@
+---
+
+# Get pipeline uuid
+- name: Get pipeline uuid
+  become: "yes"
+  command: >
+    mysql --user="{{archivematica_src_am_db_user }}"
```

Comment:
should we use dbshell here?

Reply:
It's the MCP database, so it is always MySQL. But we can use the dbshell always 👍